docs/configure_admin_workstation_post_install.rst

```diff
@@ -33,7 +33,8 @@ reboots, the Tails instance must have persistence enabled (specifically, the
 .. note:: Starting in version 0.3.7, SecureDrop requires Tails 2.x or greater.
 
 To install the auto-connect configuration, start by navigating to the directory
-with these scripts, and run the install script:
+with these scripts (``~/Persistent/securedrop/install_files/ansible-base/``),
+and run the install script:
```

Comment:
This is the path that has `securedrop-admin`. It should be `~/Persistent/securedrop`